### PR TITLE
fix(22160): temp fix to enable sending txn via QR

### LINKED ISSUE #22160: [Bug]: The scan button next to the receiving address, when tapped to scan the QR code, triggers an error.
### Describe the bug

Hope you have a great weekend, MetaMask team!
We have encountered a general issue while using the MetaMask extension. The scan button next to the receiving address, when tapped to scan the QR code, triggers an error.
Here is some information that might be helpful -
TypeError: this.codeReader.stop is not a function
  at _.teardownCodeReader (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-3.js:1:272453)
  at _.componentWillUnmount (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-3.js:1:272353)
  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:176110
  at Ss (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:176151)
  at Os (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:177912)
  at Bl (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:197202)
  at n.unstable_runWithPriority (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-2.js:1:197381)
  at Zo (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:143313)
  at Nl (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:195082)
  at Tl (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:185454)
  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:143604
  at n.unstable_runWithPriority (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-2.js:1:197381)
  at Zo (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:143313)
  at ti (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:143549)
  at ei (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:143484)
  at bl (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:182020)
  at Object.enqueueSetState (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-1.js:33:148265)
  at v.setState (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-2.js:1:176562)
  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-3.js:1:241634
  at HTMLDivElement.n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/ui-3.js:1:241349)

### Expected behavior

Able to detect the QR code and read it

### Screenshots/Recordings

https://github.com/MetaMask/metamask-extension/assets/7900773/6121fa5f-4c4e-4bbe-9f7c-b75299fa2044

_No response_

### Steps to reproduce

1. Open the MetaMask extension
2. Tap [Send] > scan button to fill the receiving address
3. Use ETH address on Keystone to let MetaMask scan it

### Error messages or log output

_No response_

### Version

11.5.1

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

Keystone

### Additional context

_No response_

### Severity

_No response_

### PR DESCRIPTION
**Pull request by DDDDDanica**
## **Description**
There was a bump for library @zxing.js https://github.com/MetaMask/metamask-extension/pull/19225 to audit the package, however, there're breaking changes from API in this library. We will have a temporary fix by reverting the upgrade and ignore the audit changes, but we will have a follow up PR to have proper fix and add e2e test to backup. 
We also include the copy update as [requested](https://github.com/MetaMask/metamask-extension/issues/22160#issuecomment-1841573078). 

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/22160

## **Manual testing steps**

1. Go to send
2. Click scanner icon beside address
3. provide a QR code to scan

### **After**

https://github.com/MetaMask/metamask-extension/assets/12678455/b792c931-4213-4e85-b376-c6cf7f5ef01a

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by DDDDDanica** [comment]
@metamaskbot update-policies

**Comment by DDDDDanica** [comment]
@metamaskbot update-policies

File: .yarnrc.yml [hunk 1]
```diff
@@ -43,6 +43,12 @@ npmAuditIgnoreAdvisories:
   # not appear to be used.
   - 1092461
 
+
+  # Temp fix for https://github.com/MetaMask/metamask-extension/pull/16920 for the sake of 11.7.1 hotfix
+  # This will be removed in this ticket https://app.zenhub.com/workspaces/extension-delivery-board-6216892781ac020010e826d1/issues/gh/metamask/metamask-extension/22299
```
Comment (danjm) [inline .yarnrc.yml:48]:
```suggestion
  # This will be removed in this ticket https://github.com/MetaMask/metamask-extension/issues/22299
```

**Review by danjm — CHANGES_REQUESTED** [review]
Looks good apart from Pedro's comment about language files and a bit more detail should be added to the PR description to explain why this change was needed and what the follow up PR will do

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -5565,10 +5565,10 @@
     "message": "Unknown"
   },
   "unknownCameraError": {
-    "message": "There was an error while trying to access your camera. Please try again..."
```
[the remark this answers is not in the document]
Comment (DDDDDanica) [reply app/_locales/en/messages.json:5568]: Super good suggestions, changes are pushed

File: .yarnrc.yml (hunk 1 above)
Comment (DDDDDanica) [reply .yarnrc.yml]: Gotcha

**Comment by DDDDDanica** [comment]
@metamaskbot update-policies

**Comment by DDDDDanica** [comment]
@metamaskbot update-policies

**Review by danjm — APPROVED** [review]

**Review by pedronfigueiredo — APPROVED** [review]

**Pull request merged**